.github/workflows/test.yml

```diff
@@ -815,6 +815,41 @@ jobs:
       matrix:
         spec:
           [
+            '.portal/cypress/e2e/journeys/*.spec.js',
+            '.portal/cypress/e2e/journeys/homepage/**/*.spec.js',
+            '.portal/cypress/e2e/journeys/admin/**/*.spec.js',
+            '.portal/cypress/e2e/journeys/checker/**/*.spec.js',
+            '.portal/cypress/e2e/journeys/dashboard/dashboard-deals/filter-by/*.spec.js',
+            '.portal/cypress/e2e/journeys/dashboard/dashboard-deals/filters/*.spec.js',
+            '.portal/cypress/e2e/journeys/dashboard/dashboard-deals/sort/*.spec.js',
+            '.portal/cypress/e2e/journeys/dashboard/dashboard-facilities/filter-by/*.spec.js',
+            '.portal/cypress/e2e/journeys/dashboard/dashboard-facilities/filters/*.spec.js',
```

Comment:
These are affected by the feature flag but it probably makes sense to test all of them to prevent any issues appearing once the FF is turned on

Reply:
the two facility filtering ones are, the rest arent